tests/generation/FileGlobs.py

```diff
@@ -64,29 +64,63 @@ def test_get_project_files_gitignore(self):
 __pycache__
 # End of gitignore""")
 
-        files = [os.path.join("src", "main.c"),
```

Comment:
Since we have the chance to fix this, ...

please restore these lines back to here, and make the minor modifications required.

There is no benefit in moving these lines around, and moving them makes it harder to see what changes are being made to these tests.

It breaks blame, and makes the patch bigger.